### ISSUE TITLE
Automatic login after entering pin

### ISSUE THREAD

**Issue by connorhsm** [voteflux/digipol#214]
I'm all about reducing that extra click.

After entering your four digit pin to log in it would be a nice to have for the login to proceed without clicking the login button.

A four digit pin is quick to enter and often top of mind, this login is also completely local on the device (afaik) so entering an incorrect pin does not waste a significant amount of time waiting on the network to attempt again.
